src/Configuration/test/Kubernetes.ServiceBinding.Test/LegacyConnectorsPostProcessorsTest.cs

```diff
@@ -11,20 +11,19 @@ namespace Steeltoe.Extensions.Configuration.Kubernetes.ServiceBinding.Test;
 public class LegacyConnectorsPostProcessorsTest : BasePostProcessorsTest
 {
     [Fact]
-    public void RabbitMQTest_BindingTypeEnabled()
+    public void MongoDbTest_BindingTypeEnabled()
     {
-        var postProcessor = new RabbitMQLegacyConnectorPostProcessor();
+        var postProcessor = new MongoDbLegacyConnectorPostProcessor();
 
-        Dictionary<string, string> configData = GetConfigData(_testBindingName, RabbitMQPostProcessor.BindingTypeKey, Tuple.Create("addresses", "test-addresses"), Tuple.Create("host", "test-host"), Tuple.Create("password", "test-password"), Tuple.Create("port", "test-port"), Tuple.Create("username", "test-username"), Tuple.Create("virtual-host", "test-virtual-host"));
+        Dictionary<string, string> configData = GetConfigData(_testBindingName, MongoDbPostProcessor.BindingTypeKey, Tuple.Create("database", "test-database"), Tuple.Create("host", "test-host"), Tuple.Create("password", "test-password"), Tuple.Create("port", "test-port"), Tuple.Create("username", "test-username"));
 
-        // BindingType enabled
-        postProcessor.PostProcessConfiguration(GetConfigurationProvider(postProcessor, RabbitMQPostProcessor.BindingTypeKey, true), configData);
-        Assert.Equal("test-addresses", configData["rabbitmq:client:uri"]);
-        Assert.Equal("test-host", configData["rabbitmq:client:server"]);
-        Assert.Equal("test-password", configData["rabbitmq:client:password"]);
-        Assert.Equal("test-port", configData["rabbitmq:client:port"]);
-        Assert.Equal("test-username", configData["rabbitmq:client:username"]);
-        Assert.Equal("test-virtual-host", configData["rabbitmq:client:virtualhost"]);
+        // BindingTypeKey enabled
```

Comment:
Why change this? It corresponds to the tests for non-legacy, which include cases for enabling/disabling the binding type.

Reply:
The diff looks like a change, but I sorted everything in both alphabetically and nothing should have changed for the RabbitMQ code